packages/headless/src/controllers/facets/range-facet/numeric-facet/headless-numeric-facet-options.ts

```diff
@@ -26,15 +25,85 @@ import {
   numberOfValues,
 } from '../../_common/facet-option-definitions';
 
-type OptionalFacetId = Partial<
-  Pick<AutomaticRangeFacetOptions<NumericFacetRequest>, 'facetId'>
->;
+export interface NumericFacetValueOption {
```

Comment:
This interface, as well as all the others here, should have a top-level doc comment explaining what they are. It took me a minute to figure out what this interface is used for.

Reply:
Could we reuse the `NumericRangeRequest` interface?